app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+from app.cafe import Cafe
+
+from typing import List
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
```

Comment:
You can use list[] instead of List in your type hint, even if you're using Python 3.9 or later